lib/tests/streamlit/runtime/runtime_test.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018-2022 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+from unittest.mock import MagicMock
+
+from streamlit.runtime.runtime import RuntimeState, SessionClient
```

Comment:
nit (and also probably worth doing in a separate PR vs in this one)

The `streamlit.runtime.runtime` feels clumsy -- maybe the `__init__.py` file in the `runtime` module should reexport things to avoid it?

Reply:
Yeah, that's a good call - I'll add it to my list of things to do before merging (so it'll land in a separate PR)